source/js/nav_jquery.js

```diff
@@ -81,17 +130,30 @@ $(document).ready(function() {
     /**
      * @description Controls the slide-in animation for the {selector}.
      * @param {*} selector
+     * @param {Boolean} isDesktop
      */
-    function slideIn(selector = searchInput) {
+    function slideIn(isDesktop, selector = searchInput) {
+        var width;
+        if (isDesktop)
+            width = "208px"; //calc(520px*0.4)
```

Comment:
as described earlier, mane teams tend to have a preference for either adding the `{` `}` or leaving these on the same line as the `if` and `else`. We can keep it this way if we much prefer it

Reply:
I don't get what you mean by this.

Reply:
what I mean is that many team would either go for:
```
if (x) doBlah()
else doBlah2()
```
or
```
if (x) {
    doBlah()
} else {
    doBlah2()
}
```

Reply:
I mean we can go with either of the two options, since there is no difference in terms of performance or functionality.

Reply:
I prefer:
```
if (x) {
    doBlah()
} else {
    doBlah2()
}
```
or just a ternary operator:
```
(x) ? doBlah() : doBlah2();
```